test/plugins.js

```diff
@@ -231,15 +247,33 @@ describe('Plugins', function () {
   describe('other options', function () {
     it('should modify options in plugin runtime', function () {
       var html = '<div class="cls"><br><rect></div>'
-      var ref = '<div class="cls"><br /><rect /></div>'
+      var ref = '<div class="cls"><br>< /><rect></rect></ /></div>'
```

Comment:
@voischev This needs to be fixed, but I don't now how and why it is affected at all ;)